src/models/Review.js

```diff
@@ -0,0 +1,33 @@
+import db from '../db';
+
+export const reviewSchema = db.Schema({
+  tutor: {
+    type: db.Schema.Types.ObjectId,
+    ref: 'User',
+    required: true,
+  },
+  learner: {
+    type: db.Schema.Types.ObjectId,
+    ref: 'User',
+    required: true,
+  },
+
+    userRating: [{
+    reviewedName: {type: String, default: ''},
```

Comment:
names can be retrieved from the reviewer and reviewee models, no need to store them directly.